processes/ingest/hathi_trust.py

```diff
@@ -36,44 +37,50 @@ def runProcess(self):
 
         logger.info(f'Ingested {len(self.records)} Hathi Trust records')
 
-    def importRemoteRecords(self, fullOrPartial=False):
```

Comment:
nit: fullOrPartial is somewhat confusing as a variable name with a boolean type. When this value is true, does that mean we are doing a full or partial import? 

Recommend changing this variable name to full like the other ingest processes.

Reply:
It is a confusing variable anytime I've looked at it and I believe it should be just full since for the complete process we are only ingesting hathifiles with a full field of true and a full field of false for the daily process, so the partial portion is unnecessary. I agree to rename the variable.

Reply:
Looking at it again I believe `True` represented full and `False` represented partial.